Tests/Functional/Frontend/TeaIndexTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Tea\Tests\Functional\Frontend;
+
+use TTN\Tea\Frontend\TeaIndex;
+use TYPO3\TestingFramework\Core\Functional\Framework\Frontend\InternalRequest;
+
+/**
+ * @coversNothing
+ */
+final class TeaIndexTest extends AbstractTest
```

Comment:
I'd suggest making this test about the whole controller, i.e., rename it to `TeaControllerTest`, move it to `Tests/Functional/Controller` and have it `@covers` the controller.